css/its-a-trap.css

```diff
@@ -11,17 +13,21 @@
     width: 430px;
 }
 
+* html #its-unwanted #main-feature,
 * html #its-a-trap #main-feature,
 * html #its-an-attack #main-feature { height: 265px; /* Hack for IE6 */ }
```

Comment:
> My understanding is that this page is invoked from inside the browser.

I'm not sure what you mean by "invoked inside the browser" but this is a page that users might be taken to from SUMO or other documentation in order to test that they have Safe Browsing turned on.